components/brave_wallet/browser/zcash/rust/zcash_primitives/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "zcash_primitives"
+version = "0.15.1"
+edition = "2021"
+license = "MPL-2.0"
+
+[dependencies]
+orchard = { version = "0.7.1",  default-features = false }
+incrementalmerkletree = { version = "0.5.1",  features=["legacy-api"] }
```

Comment:
where are you getting `legacy-api` from? I don't see that in https://github.com/brave/librustzcash/blob/zcash_primitives_for_shielding/Cargo.toml#L47C1-L47C22 and the feature is not added in the incrementalmerkletree BUILD.gn

Reply:
https://github.com/brave/librustzcash/blob/zcash_primitives_for_shielding/zcash_primitives/Cargo.toml#L55 - this is the right crate